module/Finna/src/Finna/RecordDriver/SolrEad3.php

```diff
@@ -1593,18 +1593,22 @@ public function getUnitDates()
                     [$start, $end] = explode('/', $normal);
                 } else {
                     $start = $normal;
-                    $dateParts = explode('-', $start);
-                    $yearUncertain = $this->unknownDateCharsExist($dateParts[0]);
+                    $startDateParts = explode('-', $start);
```

Comment:
Tämä olisi selkeämpi, jos laittaisi näin: `[$startYear] = explode('-', $start); `Ja vastaavasti loppuvuoden puolella, niin ei tarvitse sitten mitään parts-arrayta käyttää.